notebook/notebookapp.py

```diff
@@ -1667,10 +1672,23 @@ def init_shutdown_no_activity(self):
             pc = ioloop.PeriodicCallback(self.shutdown_no_activity, 60000)
             pc.start()
 
+    def init_eventlog(self):
+        self.eventlog = EventLog(parent=self)
+
+        yaml = YAML(typ='safe')
```

Comment:
Can we drop this call if it's already inside jupyter/telemetry (see [here](https://github.com/jupyter/telemetry/blob/0022131cefac065909fcbb85a5c2b2bf3a6adc8a/jupyter_telemetry/eventlog.py#L16))?

Reply:
Yep, we do that in jupyter_telemetry now, this PR needs updating